### PR TITLE
Update alpine 3.8

### PR DESCRIPTION
**Pull request by marians**
For https://github.com/giantswarm/giantswarm/issues/4197

### CODE REVIEW

**Review by MarcelMue — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+version: 2
+jobs:
+  build:
+    machine: true
+    environment:
+      - FOO: "foo"
```
Comment (MarcelMue) [inline .circleci/config.yml]: Why are these here?

Comment (marians) [reply .circleci/config.yml]: That's stupid copy-pasting from our Wiki
